dds/DCPS/XTypes/TypeObject.h

```diff
@@ -217,7 +217,7 @@ namespace XTypes {
   struct EquivalenceHash_tag {};
   typedef ACE_CDR::Octet EquivalenceHash_slice;
   typedef Fake_TAO_Array_Forany_T<EquivalenceHash, EquivalenceHash_slice, EquivalenceHash_tag> EquivalenceHash_forany;
-  const DCPS::String equivalence_hash_to_string(const EquivalenceHash& hash);
+  OpenDDS_Dcps_Export const DCPS::String equivalence_hash_to_string(const EquivalenceHash& hash);
```

Comment:
Thanks Clayton. I keep forgetting to add the export macro and was wondering why it didn't link.